packages/fast-tooling-react/src/style/utilities.ts

```diff
@@ -51,6 +51,31 @@ export function applyTriggerStyle(color: string): CSSRules<{}> {
     };
 }
 
+export function applyInteractiveFormItemIndicator(): CSSRules<{}> {
```

Comment:
Can this just be an object?

Reply:
Actually, good point, these can all just be objects, I filed a task to convert all of them here: https://github.com/Microsoft/fast-dna/issues/1692